test/collections.statistics.js

```diff
@@ -0,0 +1,87 @@
+      assert.strictEqual(_.standardDeviation([1, 2, 3, 4]), 1.118033988749895, "Standard Deviation of the numbers in the collection");
+      assert.strictEqual(_.standardDeviation({}), 0, 'Standard Deviation value of an empty object');
+      assert.strictEqual(_.standardDeviation([]), 0, 'Standard Deviation value of an empty array');
+    });
+  
+    QUnit.test('standardError', function(assert) {
+      assert.strictEqual(_.standardError(null), 0, 'can handle null/undefined');
+      assert.strictEqual(_.standardError(void 0), 0, 'can handle undefined');
+      assert.strictEqual(_.standardError([0, 1, 2, 3, 4]), 0.7071067811865476, "Standard Error of the numbers in the collection");
+      assert.strictEqual(_.standardError([1, 2, 3, 4]), 0.6454972243679028, "Standard Error of the numbers in the collection");
+      assert.strictEqual(_.standardError({}), 0, 'Standard Error value of an empty object');
+      assert.strictEqual(_.standardError([]), 0, 'Standard Error value of an empty array');
+    });
+  
+    QUnit.test('mode', function(assert) {
+      assert.strictEqual(_.mode(null), undefined, 'can handle null/undefined');
+      assert.strictEqual(_.mode(void 0), undefined, 'can handle undefined');
+      assert.strictEqual(_.mode([0, 1, 2, 3, 4]), 0, "Mode of the numbers in the collection");
+      assert.strictEqual(_.mode([1, 1, 3, 4]), 1, "Mode of the numbers in the collection");
+      assert.strictEqual(_.mode({}), undefined, 'Mode value of an empty object');
+      assert.strictEqual(_.mode([]), undefined, 'Mode value of an empty array');
+    });
+  
+    QUnit.test('statRange', function(assert) {
+      assert.strictEqual(_.statRange(null), -Infinity, 'can handle null/undefined');
+      assert.strictEqual(_.statRange(void 0), -Infinity, 'can handle undefined');
+      assert.strictEqual(_.statRange([0, 1, 2, 3, 4]), 4, "Stat Range of the numbers in the collection");
+      assert.strictEqual(_.statRange([1, 1, 3, 4]), 3, "Stat Range of the numbers in the collection");
+      assert.strictEqual(_.statRange({}), -Infinity, 'Stat Range value of an empty object');
+      assert.strictEqual(_.statRange([]), -Infinity, 'Stat Range value of an empty array');
+    });
+    
+    QUnit.test('percentile', function(assert) {
+      assert.strictEqual(_.percentile(null, 25), 0, 'can handle null/undefined');
+      assert.strictEqual(_.percentile(void 0, 50), 0, 'can handle undefined');
+      assert.strictEqual(_.percentile([0, 1, 2, 3, 4], 75), 3, "75th percentile of the numbers in the collection");
+      assert.strictEqual(_.percentile([1, 1, 3, 4], 50), 2, "50th of the numbers in the collection");
+      assert.strictEqual(_.percentile({}, 10), 0, 'Percentile value of an empty object');
+      assert.strictEqual(_.percentile([], 50), 0, 'Percentile value of an empty array');
+      assert.raises(function() {
+        _.percentile([1, 1, 3, 4], "50");
+      }, TypeError, 'Percentile must be a number between 0 - 100');
+   });
+  }());
```

Comment:
Matching fix at the end

```suggestion
    assert.strictEqual(_.percentile([], 50), 0, 'Percentile value of an empty array');
    assert.raises(function() {
      _.percentile([1, 1, 3, 4], "50");
    }, TypeError, 'Percentile must be a number between 0 - 100');
   });
  }());
```